main.cpp

```diff
@@ -546,13 +436,128 @@ MOVE THEM to the space below this block comment and put them in numerical order
 */
 
 
+/*
 
+Thing 1) TV
+5 properties:
+    1) broadcasted channel (int)
+    2) volume level (int)
+    3) amount of display ports (int)
+    4) internet connection (bool)
+    5) amount of smartTV apps (number)
+3 things it can do:
+    1) increase/decrease volume
+    2) select specific channel to be broadcast
+    3) change between display ports
 
+Thing 2) Laptop
+5 properties:
+    1) screen display brightness (int)
+    2) keyboard lights (bool)
+    3) mousepad sensitivity (int)
+    4) battery left percentage (int)
+    5) amount of USB ports (int)
```

Comment:
Be more creative than just `int` for all of your properties of this laptop.
how about 'cost/brand/model number' or 'production date (std::string)'?
- [ ] revise